Source/WebCore/html/shadow/spatialImageControls.css

```diff
@@ -81,6 +81,7 @@ div#bottom-gradient {
     left: 0;
     height: 33%;
     width: 100%;
+    border-radius: inherit;
```

Comment:
doesn't it look weird if we inherit big radii on the top corners? (since this is not full height)
maybe we can set only the bottom corners to inherit